test/topology_experimental_raft/test_raft_service_levels.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import pytest
```

Comment:
I will but I need to find out why the test sometimes fails when I move it to `auth_cluster`.

I know where because it fails when I'm trying to connect to newly created node but it's completely fine is `topology_experimental_raft` suite. Putting sleep() after adding the new node increased ratio of passed tests but adding sleep isn't a solution unfortunately.

Reply:
I'll move it after rebasing on auth-on-raft. `auth_cluster` suite requires authentication but without auth-on-raft it cannot be done without quorum.